app/templates/gulp/gulp/tasks/copy.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var $ = require( 'gulp-load-plugins' )();
```

Comment:
https://github.com/cfpb/cfgov-refresh/issues/830 could be updated here or later when 830 is closed.